Models/CandidateTestModel.ts

```diff
@@ -0,0 +1,20 @@
+export const DefaultExecutionTimeLimit = "4 seconds";
+export const DefaultJsTips = "// Prints help message to the console\n" + "Returns a string\n" + "function helloWorld(name) {\n" + "    console.log(\"This prints to the console when you Run Tests\");\n" + "    return \"Hello, \" + name;\n" + "}";
```

Comment:
I think we can now remove these

Reply:
I was going to leave them there for future use?

Reply:
I removed them